### PR TITLE
Version bump to v2021.0.3

### PR DESCRIPTION
**Pull request by turian**
Includes #59 #60 and #61

We switch to an 80/10/10 split

### CODE REVIEW

File: hearpreprocess/pipeline.py [hunk 1]
```diff
@@ -723,7 +726,7 @@ def run(self):
         if max_split_duration is None:
             max_files = len(split_filestem_relpaths)
         else:
-            max_files = int(MAX_TASK_DURATION_BY_SPLIT[self.split] / sample_duration)
+            max_files = int(max_split_duration / sample_duration)
```
Comment (turian) [inline hearpreprocess/pipeline.py:729]: This was a bug I didn't catch until now

File: hearpreprocess/util/luigi.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def workdir(self):
     @property
     def task_subdir(self):
         """Task specific subdirectory"""
-        return Path(self.task_config.get("luigi_dir", "_workdir")).joinpath(
+        return Path(self.task_config.get("tmp_dir", "_workdir")).joinpath(
```
Comment (turian) [inline hearpreprocess/util/luigi.py:89]: This was a bug I didn't catch until now

File: README.md [hunk 3]
```diff
@@ -64,8 +66,10 @@ If you want to run preprocessing yourself:
 libsox-fmt-ffmpeg or [installing from
 source](https://github.com/neuralaudio/hear-eval-kit/issues/156#issuecomment-893151305).
 
-This will take about 2 user-CPU-hours for the open tasks. 150 GB free
-disk space is required while processing. Final output is 11 GB.
+This will take about 2 user-CPU-hours for the open tasks, when using
+mode 'default'. 150 GB free disk space is required while processing.
+Final output is 11 GB. Much more time and disk space is required
+for 'all'.
```
Comment (jorshi) [inline README.md:72]: We're now using 'all' for two args (task selection and the mode) -- let's make it really clear which one we are referring to here.

File: hearpreprocess/nsynth_pitch.py [hunk 4]
```diff
@@ -48,16 +54,16 @@
     "modes": {
         "5h": {
             "max_task_duration_by_split": {
-                "train": 3600 * 5 * 3 / 4,
-                "valid": 3600 * 5 * 1 / 4,
-                "test": 3600 * 5 * 1 / 4,
+                "train": 3600 * 5 * TRAIN_PERCENTAGE / TRAINVAL_PERCENTAGE,
+                "valid": 3600 * 5 * VALIDATION_PERCENTAGE / TRAINVAL_PERCENTAGE,
+                "test": 3600 * 5 * TEST_PERCENTAGE / TRAINVAL_PERCENTAGE,
```
Comment (jorshi) [inline hearpreprocess/nsynth_pitch.py:59]: I'm confused. These percentages add up to over 100%. If I understand, we are saying the train and valid should add up to 5hrs? Maybe this should be documented.

File: hearpreprocess/pipeline.py [hunk 5]
```diff
@@ -27,22 +27,25 @@
...
 # the data in the split and hence is not a part of the data config
-VALIDATION_PERCENTAGE = 20
-TEST_PERCENTAGE = 20
+# We use a 80/10/10 split.
+VALIDATION_PERCENTAGE = 10
+TEST_PERCENTAGE = 10
 TRAIN_PERCENTAGE = 100 - VALIDATION_PERCENTAGE - TEST_PERCENTAGE
+TRAINVAL_PERCENTAGE = TRAIN_PERCENTAGE + VALIDATION_PERCENTAGE
 
 # We want no more than 5 hours of audio (training + validation) per task.
```
Comment (jorshi) [inline hearpreprocess/pipeline.py:41]: Ah I see.

**Review by jorshi — APPROVED** [review]

**Pull request merged**